libs/portal-plugin-dashboard/src/capabilities/refineConfig.ts

```
@@ -27,33 +28,44 @@ export class Capability implements RefineConfigCapability {
+
     const mergedConfig = {
       ...existing,
+      dataProvider: {
+        ...normalizedDataProvider,
+        [DATA_PROVIDER_NAME]: acctProvider,
+      },
       options: {},
       resources: [
-        {
-          name: "account",
+        ...(mergedConfig.resources || []),
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)